server/src/models/User.js

```diff
@@ -43,6 +45,33 @@ class User extends uniqueFunc(Model) {
 
     return serializedJson;
   }
+
+  static get relationMappings() {
+    const { Review, Activity } = require("./index.js")
+    return {
+      activities: {
```

Comment:
are we using this association? do we need to see the list of activities the user left a review on? most likely we would want to see all the reviews that a user left (and can see the activity they left a review on that way)